test/SILGen/non_loadable_move_only.swift

```diff
@@ -0,0 +1,18 @@
+// RUN: %target-swift-emit-silgen -module-name=test -primary-file %s | %FileCheck %s
+
+@_moveOnly
+public struct GenericMoveOnly<T> {
+  var i: Int
+  var s: T
+
+  // CHECK-LABEL: sil [ossa] @$s4test15GenericMoveOnlyVfD : $@convention(method) <T> (@in GenericMoveOnly<T>) -> ()
```

Comment:
This is more of a meta question. Do we have any Interpreter tests that test this out? It would be good to add this.